prow/pubsub/subscriber/subscriber.go

```diff
@@ -327,7 +331,10 @@ func (s *Subscriber) handleMessage(msg messageInterface, subscription string, al
 	eType, err := extractFromAttribute(msg.getAttributes(), prowEventType)
 	if err != nil {
 		l.WithError(err).Error("failed to read message")
-		s.Metrics.ErrorCounter.With(prometheus.Labels{subscriptionLabel: subscription})
+		s.Metrics.ErrorCounter.With(prometheus.Labels{
```

Comment:
I don't think this actually increments the counter (and didn't previously either). We need to append a `.Inc()` call to actually use the counter that we've just indexed out of the vector. This applies to the other places as well.

Reply:
good catch, fixed